6-game-of-life/game.rb

```diff
@@ -0,0 +1,73 @@
+class Game
```

Comment:
Rubocopowe uwagi co do stylu:

```
game.rb:1:1: C: Style/Documentation: Missing top-level class documentation comment.
class Game
^^^^^
game.rb:18:10: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
    puts "---------------"
         ^^^^^^^^^^^^^^^^^
game.rb:21:3: C: Metrics/AbcSize: Assignment Branch Condition size for check_field is too high. [39.1/15]
  def check_field(field) ...
  ^^^^^^^^^^^^^^^^^^^^^^
game.rb:21:3: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for check_field is too high. [7/6]
  def check_field(field) ...
  ^^^^^^^^^^^^^^^^^^^^^^
game.rb:21:3: C: Metrics/MethodLength: Method has too many lines. [16/10]
  def check_field(field) ...
  ^^^^^^^^^^^^^^^^^^^^^^
game.rb:21:3: C: Metrics/PerceivedComplexity: Perceived complexity for check_field is too high. [8/7]
  def check_field(field) ...
  ^^^^^^^^^^^^^^^^^^^^^^
game.rb:26:18: C: Layout/SpaceAroundOperators: Surrounding space missing for operator -.
    options = [[x-1, y-1], [x-1, y], [x-1, y+1], [x, y-1],
                 ^
game.rb:26:23: C: Layout/SpaceAroundOperators: Surrounding space missing for operator -.
    options = [[x-1, y-1], [x-1, y], [x-1, y+1], [x, y-1],
                      ^
game.rb:26:30: C: Layout/SpaceAroundOperators: Surrounding space missing for operator -.
    options = [[x-1, y-1], [x-1, y], [x-1, y+1], [x, y-1],
                             ^
game.rb:26:40: C: Layout/SpaceAroundOperators: Surrounding space missing for operator -.
    options = [[x-1, y-1], [x-1, y], [x-1, y+1], [x, y-1],
                                       ^
game.rb:26:45: C: Layout/SpaceAroundOperators: Surrounding space missing for operator +.
    options = [[x-1, y-1], [x-1, y], [x-1, y+1], [x, y-1],
                                            ^
game.rb:26:55: C: Layout/SpaceAroundOperators: Surrounding space missing for operator -.
    options = [[x-1, y-1], [x-1, y], [x-1, y+1], [x, y-1],
                                                      ^
game.rb:27:7: C: Layout/AlignArray: Align the elements of an array literal if they span more than one line.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
      ^^^^^^^^
game.rb:27:12: C: Layout/SpaceAroundOperators: Surrounding space missing for operator +.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
           ^
game.rb:27:19: C: Layout/SpaceAroundOperators: Surrounding space missing for operator +.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
                  ^
game.rb:27:24: C: Layout/SpaceAroundOperators: Surrounding space missing for operator -.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
                       ^
game.rb:27:31: C: Layout/SpaceAroundOperators: Surrounding space missing for operator +.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
                              ^
game.rb:27:41: C: Layout/SpaceAroundOperators: Surrounding space missing for operator +.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
                                        ^
game.rb:27:46: C: Layout/SpaceAroundOperators: Surrounding space missing for operator +.
      [x, y+1], [x+1, y-1], [x+1, y], [x+1, y+1]]
                                             ^
game.rb:36:32: C: Layout/SpaceBeforeBlockBraces: Space missing to the left of {.
    positive = neighbours.count{ |el| el == 1 }
                               ^
game.rb:38:8: C: Style/NumericPredicate: Use field_type.zero? instead of field_type == 0.
    if field_type == 0 && positive == 3
       ^^^^^^^^^^^^^^^
game.rb:39:19: C: Layout/SpaceInsideHashLiteralBraces: Space inside { missing.
      @to_live << {x: x, y: y}
                  ^
game.rb:39:30: C: Layout/SpaceInsideHashLiteralBraces: Space inside } missing.
      @to_live << {x: x, y: y}
                             ^
game.rb:41:18: C: Layout/SpaceInsideHashLiteralBraces: Space inside { missing.
      @to_die << {x: x, y: y }
                 ^
game.rb:43:1: C: Layout/EmptyLinesAroundMethodBody: Extra empty line detected at method body end.
game.rb:48:30: W: Lint/UnusedBlockArgument: Unused block argument - field. If it's necessary, use _ or _field as an argument name to indicate that it won't be used.
      row.each_with_index { |field, field_index| check_field({ x: row_index, y: field_index }) }
                             ^^^^^
game.rb:48:62: C: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
      row.each_with_index { |field, field_index| check_field({ x: row_index, y: field_index }) }
                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
game.rb:48:81: C: Metrics/LineLength: Line is too long. [96/80]
      row.each_with_index { |field, field_index| check_field({ x: row_index, y: field_index }) }
```